src/utils/__tests__/logger.test.js

```diff
@@ -1,12 +1,22 @@
 import SumoLogger from 'sumo-logger';
-import logger from '../logger';
+import logger, { Logger } from '../logger';
 
 jest.mock('sumo-logger');
 
 describe('Logger', () => {
+  afterEach(() => {
+    SumoLogger.mockClear();
+  });
+
   test('sends message to logger', () => {
     logger.log('test message');
     expect(SumoLogger).toHaveBeenCalled();
     expect(SumoLogger.mock.instances[0].log).toHaveBeenCalledWith('test message');
   });
+
+  test('does nothing when no endpoint is provided', () => {
+    const localLogger = new Logger('');
```

Comment:
Instead of having multiple exports from the file just for testing, could `global.LOGGER_ENDPOINT = null;` work here?

Reply:
I played with other options for a while, and I think exposing the constructor is the cleanest way to allow testing it, since there is some important logic in the constructor. Otherwise, the test would need to find a way to re-import the logger after setting the endpoint to null, since the constructor is currently called on import.

Philosophically, I kind like exposing the constructor, too; it would allow somebody to use it more functionally and supply their own endpoint. (Not that I imagine any of our components will want a logger with some other endpoint :shrug:)

Reply:
Ok. Makes sense